README.md

```diff
@@ -53,20 +53,33 @@ All commands are run from the root of the project, from a terminal:
 | `npm run textlint`     | Run textlint on content collections                |
 | `npm run textlint:fix` | Apply fixable updates to resolve texlint errors    |
 
+
+
 ### Local Development Setup
 
+1. Run `cp .env.example .env` to create a `.env` file for environment variables. (Don’t check this in!)
+2. Create a [classic GitHub access token](https://github.com/settings/tokens) with these scopes: `repo`, `read:org`, `read:user`, and `read:project`.
+3. Paste the GitHub token after `.env`’s `GITHUB_TOKEN=`.
+
+#### DDev setup
+
+DDev already has all the dependencies included.
+
+1. Run `ddev start && ddev npm install` set up the project’s dependencies.
+3. `npm run build` can be found at the base URL and `npm run dev` is found at URL:4321. Dev has Vite HMR(hot module reloading) among other features. The site will automatically refresh as you work on it, displaying errors in the relevant console (terminal or browser).
```

Comment:
```suggestion
2. `ddev npm run build` can be found at the base URL and `ddev npm run dev` is found at URL:4321. Dev has Vite HMR(hot module reloading) among other features. The site will automatically refresh as you work on it, displaying errors in the relevant console (terminal or browser).
```

Reply:
${DDEV_PRIMARY_URL}:4321 -  I was looking for that variable for a while haha, I will work on that.

http://127.0.0.1/ is for the local setup without DDev in case they just want to run npm but maybe I can make that clearer I will double check.

I did not see that build error I will check on that one as well.

Thanks for the feedback.